migrations/versions/0397_missed_steps.py

```diff
@@ -0,0 +1,21 @@
+"""
+
+Revision ID: 0397_missed_steps
+Revises: 0396_make_research_mode_nullable
+Create Date: 2023-01-19 15:56:28.950942
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+revision = "0397_missed_steps"
+down_revision = "0396_make_research_mode_nullable"
+
+
+def upgrade():
+    # This was created as `sa.String` in the migration but defined in Python as `db.Text`.
+    op.alter_column("broadcast_message", "content", existing_type=sa.TEXT(), type_=sa.String(), existing_nullable=False)
```

Comment:
given text and varchar without a length are functionally equivalent in postgres, i think i'd rather update the model to use `db.Text` and save on doing any migration 🤔. That feels less risky (not that I'm worried about this particular alter statement especially).

(really i'd rather we unified all our usage of `db.String` (~73), `db.String(255)` (~42), and `db.Text` (~22) and just chose one approach, probably `db.Text` for anything unbounded and `db.String(n)` for stuff we want to explicitly guard against, which at the mo is some but not all mobile number fields and the hex colour for meail branding